Rewrite footnote into table introduction (#2342)
theforeman/foreman-documentation: guides/common/modules/ref_predefined-roles.adoc

[id="Predefined_Roles_{context}"]
= Predefined Roles Available in {Project}

> xref:table_permissions-provided-by-role[] provides an overview of permissions that predefined roles in {Project} grant to a user.

Reviewer: ```suggestion
The following table provides an overview of permissions that predefined roles in {Project} grant to a user.
```

To view the exact set of permissions a predefined role grants, display the role in {ProjectwebUI} as the privileged user.
For more information, see xref:Viewing_Permissions_of_a_Role_{context}[].

.Permissions provided by role

> [id="table_permissions-provided-by-role"]

Reviewer: ```suggestion
```
